CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
 # Unreleased
+- On Unix, X11 and Wayland are now optional features (enabled by default)
```

Comment:
in general `macOS` is unix(and macOS even has Xorg), so maybe `On Unix non macOS`. You should likely update it as well on that PR.

Reply:
I think it's fine the way it is. It may be technically incorrect, but "Unix" is used elsewhere in winit docs to mean "Unix except for MacOS." I think users reading the changelog will understand that X11 and Wayland are not part of the Mac winit platform.

Reply:
I kind of agree here tbqh, since winit already using not entirely accurate words, so let's just keep it the way it is.